website/docs/docs/build/packages.md

```diff
@@ -205,8 +205,11 @@ packages:
   # use this format when accessing your repository via a github application token
   - git: "https://{{env_var('DBT_ENV_SECRET_GIT_CREDENTIAL')}}@github.com/dbt-labs/awesome_repo.git" # git HTTPS URL
 
-  # use this format when accessing your repository via a personal access token
+  # use this format when accessing your repository via a classical personal access token
   - git: "https://{{env_var('DBT_ENV_SECRET_GIT_CREDENTIAL')}}@github.com/dbt-labs/awesome_repo.git" # git HTTPS URL
+ 
+   # use this format when accessing your repository via a fine-grained personal access token (username sometimes required)
+  - git: "https://<username>:{{env_var('DBT_ENV_SECRET_GIT_CREDENTIAL')}}@github.com/dbt-labs/awesome_repo.git" # git HTTPS URL
```

Comment:
@patkearns10 confirming you mean github_username here - just making it extra clear for users:

```suggestion
  - git: "https://GITHUB_USERNAME:{{env_var('DBT_ENV_SECRET_GIT_CREDENTIAL')}}@github.com/dbt-labs/awesome_repo.git" # git HTTPS URL
```